template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+- **Working groups** - Our project is organized into several different working groups that meet on a regular basis to work on the project’s key initiatives. One of the best ways to get started with our project is to join and meet with one of our working groups. See [The Good Docs Project Working Groups](https://thegooddocsproject.dev/working-group/) for a list of our current active groups. NOTE: If you plan to contribute to our project by writing templates, you will be required to join one of the template-focused working groups.
+- **Weekly meetings** - The project leaders hold two weekly meetings to discuss project-level decisions. Feel free to join one of these meetings to introduce yourself to the project leaders and discover next steps for getting involved in the project. We currently hold meetings for the U.S./Canada and APEC/EU/Africa time zones. After introducing yourself on Slack, one of the members of the welcoming committee can help you get invited to the best meeting for your time zone.
+
+As you begin to join our project, remember that this is a project composed entirely of volunteers.
+We love to welcome new members, but want to be careful not to burn out our core project contributors.
+This level of mindfulness helps us ensure that we retain our project’s capacity to produce high-quality work.
+As such, we ask that you respect the time of our project maintainers and contributors.
+(And expect us to respect your time in return!)
+
+We expect all members of our project to be nice to each other and to follow our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/) when interacting with other members of the Good Docs Project.
+
+
+## Adopt a template
+
+In this phase, you will decide which template project you will work on and adopt that template.
+
+Be aware that:
+- Each template project is represented by a corresponding issue in the [Good Docs Project templates](https://github.com/thegooddocsproject/templates/issues) repository.
+- You will use this issue to communicate the status of your template project as it moves through the different phases of the template writing process.
+- The Good Docs Project managers use a kanban board that shows all the issues for the current template projects. This tool allows the templates coordinator, template mentors, community reviewers, and other stakeholders to track the overall progress of each template and assist templateers whose progress is blocked.
+
+To adopt a template:
+
+1. Scroll through the list of template issues and see if one interests you and/or matches your skill set. Alternatively, if you have an idea for a template project that does not yet have an issue, and you have the support of a template mentor, you can create a new issue for your template project.
+
+2. Assign yourself to the issue. See Assigning issues and pull requests in the GitHub docs for more information.

```

Comment:
Wouldn't hurt to add the link: https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users

Reply:
Good call. I'll add that.

Reply:
I've adopted this suggestion. Resolving.